### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by oxk4r01** [MichaelCurrin/auto-commit-msg#89]
Mine, seems to have stopped working. When I click the button nothing happens.

**Comment by MichaelCurrin** [comment]
Can you share details? I'm using v0.25.0 on Windows as the latest one and it still works.

**Comment by bobfirestone** [comment]
It has stopped working for me also on 2 computers. Both are MacOS.

Version: 1.74.0 (Universal)
Commit: 5235c6bb189b60b01b1f49062f4ffa42384f8c91
Date: 2022-12-05T16:43:37.594Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 21.6.0
Sandboxed: No

**Comment by oxk4r01** [comment]
Here Windows 11. And comment button doesn't do anything now.

**Comment by dylan-k** [comment]
@MichaelCurrin Same here. Installed the extension @ latest version, but the button does nothing.
Windows 10. VSCode 1.7.4.0

**Comment by CosmoFox** [comment]
SAme problem with  v0.25.0
on Win11 and Ubu22.04 server

**Comment by CosmoFox** [comment]
@MichaelCurrin Please, comeback =)

**Comment by MichaelCurrin** [comment]
Ah thanks all for the info. And the GitLens tipoff.

**Comment by MichaelCurrin** [comment]
I'm going to install GitLens and check the error log

**Comment by MichaelCurrin** [comment]
Aha. Here is error message when using the sandboxed extension for log messages and GitLens installed on and kept enabled.

![image](https://user-images.githubusercontent.com/18750745/209863006-23fd7370-4719-49a5-8371-d2640d5aa882.png)

**Comment by MichaelCurrin** [comment]
Okay I have a fix to use `rootUri` instead of `_rootUri` and that works in the sandbox.

I have an issue getting the extension to install in VS Code though - my first time installing it using WSL on Windows.
```
 DONE  Packaged: build/auto-commit-msg-0.25.1.vsix (26 files, 34.73KB)
<3>init: (20977) ERROR: UtilConnectToInteropServer:307: connect failed 2
                                                                        make: *** [Makefile:52: e] Error 1
```

**Comment by MichaelCurrin** [comment]
Let me know if you have any ideas there, otherwise I'll keep testing in the sandboxed window and publish that ASAP

**Comment by MichaelCurrin** [comment]
Fixed in PR - for review if anyone wants to help 

https://github.com/MichaelCurrin/auto-commit-msg/pull/91/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626L41

**Comment by CosmoFox** [comment]
> Fixed in PR - for review if anyone wants to help
> 
> https://github.com/MichaelCurrin/auto-commit-msg/pull/91/files#diff-04bba6a35cad1c794cbbe677678a51de13441b7a6ee8592b7b50be1f05c6f626L41

out of my knowledge. Hope anyone here can help you. And thx you for your work under that project!

**Comment by MichaelCurrin** [comment]
Published as v0.25.2 in the extension store. Release on GH: https://github.com/MichaelCurrin/auto-commit-msg/releases/tag/v0.25.2

**Comment by MichaelCurrin** [comment]
I also added a feature for one than one repo to be used at once v0.26.0
